src/main/resources/cache.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I haven't. How do I set the class path when I run an app using `mvn jetty:run` ?